rules-engine/src/rules_engine/engine.py

```diff
@@ -124,6 +156,33 @@ def calculate_balance_point_and_ua(
 
             self.refine_balance_point(next_balance_point_sensitivity)
 
+
+    def calculate_balance_point_and_ua_customizable(
+        self, 
+        bps_to_remove: List, # List[BillingPeriod]
```

Comment:
`nitpick`: we could probably type this as `List[BillingPeriod]` if that's what it represents